recipe/meta.yaml

```diff
@@ -114,7 +114,7 @@ test:
     # extra f2py tests need a fortran compiler
     - {{ compiler('c') }}        # [unix]
     - {{ compiler('cxx') }}      # [unix]
-    - {{ compiler('fortran') }}  # [linux]
```

Comment:
Automatic removal of the linux-only condition may be a bug in the migrator?

Reply:
It's intentional in most cases (e.g. those that had disabled fortran on windows or had done manual workarounds for windows), but not necessarily in this one.

I guess the tests needing a Fortran compiler won't trigger based on the presence of flang? Might be interesting to see if those tests pass also with flang, but it's not essential here.

Reply:
xref gh-272, where the Fortran compiler was added. No real discussion on `# [linux]` vs. other platforms as far as I can tell. The tests themselves are pretty fragile and slow, and the `f2py` output is platform-independent, so better to leave this as is in my opinion.

Reply:
I tend to prefer more coverage over less (I think at the time I just didn't consider running fortran tests anywhere but on linux). If the test suite passes in the presence of a fortran compiler, even better IMO.